Resources/Prototypes/Reagents/fun.yml

```diff
@@ -166,3 +166,42 @@
       effects:
       - !type:Electrocute
         probability: 0.5
+
+- type: reagent
+  id: Razorium
+  name: reagent-name-razorium
+  group: Toxins
+  slippery: true
+  desc: reagent-desc-razorium
+  physicalDesc: reagent-physical-desc-reflective
+  flavor: sharp
+  color: "#e3fffb"
+  reactiveEffects:
+    Acidic:
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        scaleByQuantity: false
+        ignoreResistances: true
+        damage:
+          types:
+            Slash: 5
```

Comment:
okay i turned it down a good bit because it did turn out to be a killing machine due to dealing slash inside and out at the same time. now you can survive if you shiftwalk out within the first like 5-7 seconds or so.